tests/test_data/references/fns.txt

```diff
@@ -0,0 +1,55 @@
+//! > Test references of function.
```

Comment:
I think base case should have examples:
- 0 usages
- multiple usages in the same line